api/src/main/java/com/epam/esm/handler/InvalidFieldValueExceptionHandler.java

```diff
@@ -0,0 +1,31 @@
+package com.epam.esm.handler;
+
+import com.epam.esm.exception.InvalidFieldValueException;
+import com.epam.esm.entity.ResponseExceptionEntity;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+
+/**
+ * The InvalidFieldValueException handler.
+ *
+ * @author Yauheni Tstiov
+ */
+@ControllerAdvice
+public class InvalidFieldValueExceptionHandler {
```

Comment:
There is no need to create a separate class for each exception, you could use one class with several exception handler methods.
Also, I can't see any global exception handler (handler that catches exceptions that were not caught by all other exception handlers) just for the cases if an unexpected error occurs.